### DISCUSSION TITLE
無法啓動be

### DISCUSSION THREAD

**Discussion by moemoequte** [apache/doris#33171] [A - General / Q&A] [answered]
java：
```
openjdk version "1.8.0_402"
OpenJDK Runtime Environment (build 1.8.0_402-8u402-ga-6-b06)
OpenJDK 64-Bit Server VM (build 25.402-b06, mixed mode)
```
gcc:
```
gcc version 13.2.0 (Debian 13.2.0-23)
```
arch: x86_64
linux: Debian GNU/Linux 12 (bookworm)
cpu: 16cores
mem: 64g
disk: 500g

按照官方文檔可以正常啓動fe，并且一切參數正常。 但是be啓動以後看不到pid， 沒有報錯也沒有任何輸出， ps也不知道是哪個進程
![image](https://github.com/apache/doris/assets/70629889/1509bacc-2e5a-43b2-920e-1fcb0cbfd19d)

**Comment by yagagagaga** [comment, marked as answer]
应该是启动失败了，看下log目录的日志
